manifests/init.pp

```diff
@@ -93,6 +93,9 @@
 #   }
 #
 class postfix (
+  Stdlib::Absolutepath            $confdir,
+  Boolean                         $manage_mailname,
```

Comment:
Could you put this together with the other `manage_*` boolean parameters below, and default to `true` here instead of in `common.yaml` (just for the sake of homogeneity)?

Reply:
What about `$manage_aliases` and `$manage_root_alias` ? :trollface: